thirdTask/thirdTask/ViewController.swift

```diff
@@ -24,54 +24,83 @@ class ViewController: UIViewController, UITableViewDataSource, UITableViewDelega
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        // Set the delegate and data source of the table view
         colorsTable.delegate = self
         colorsTable.dataSource = self
+        colorsTable.register(UITableViewCell.self, forCellReuseIdentifier:  colorCell)
+        
+        setupConstraints()
         
-        // Register a basic cell type
-        colorsTable.register(UITableViewCell.self, forCellReuseIdentifier: "colorCell")
-            }
+    }
+    
+    override func viewWillLayoutSubviews() {
```

Comment:
set title property for the view controller inside viewDidLoad and show nav bar in both cases, no need for this